tinkerpop3/src/main/java/io/shiftleft/overflowdb/OdbGraph.java

```diff
@@ -137,10 +138,25 @@ private void initElementCollections(OdbStorage storage) {
     }
 
     currentId.set(maxId + 1);
+    initializeStoredIndices(storage);
     long elapsedMillis = System.currentTimeMillis() - start;
     logger.info("initialized " + this.toString() + " from existing storage in " + elapsedMillis + "ms");
   }
 
+  private void initializeStoredIndices(OdbStorage storage) {
+    storage
+        .getIndexNames()
+        .stream()
+        .forEach(indexName -> loadIndex(indexName, storage));
+  }
+
+  private void loadIndex(String indexName, OdbStorage storage) {
+    assert indexName != null;
```

Comment:
You're probably aware, but asserts in java aren't checked at runtime unless the `-ea` flag is set.
I'd just drop that line - if anyone passes `null` they deserve to suffer.